cpp/velox/compute/WholeStageResultIterator.h

```diff
@@ -69,6 +70,8 @@ class WholeStageResultIterator : public ColumnarBatchIterator {
   /// A map of custom configs.
   std::unordered_map<std::string, std::string> confMap_;
 
+  const SparkTaskInfo taskInfo_;
```

Comment:
What is main purpose to keep the task info? Seems previous temp variable is enough.

Reply:
It help use task info in other place not only in ctor.